packaging/windows/clisdk/registrykeys.wxs

```diff
@@ -10,7 +10,7 @@
       </Component>
       <Component Id="SetupRegistry_x86_RC2_Compat_Key" Directory="TARGETDIR" Win64="no">
         <RegistryKey Root="HKLM" Key="SOFTWARE\dotnet\Setup\InstalledVersions\$(var.Platform)\sdk">
-          <RegistryValue Action="write" Name="1.0.0-preview2" Type="integer" Value="1" KeyPath="yes"/>
+          <RegistryValue Action="write" Name="1.0.2-preview2" Type="integer" Value="1" KeyPath="yes"/>
```

Comment:
@livarcocc - Isn't this the SDK version string? If so, it should remain 1.0.0-preview2.